### ISSUE TITLE
object-literal-sort-keys doesn't flag errors on Windows starting in version 5.8.0

### ISSUE THREAD

**Issue by interpfister** [palantir/tslint#3426]
### Bug Report

- __TSLint version__: 5.8.0
- __TypeScript version__:2.5.3
- __Running TSLint via__: CLI and VSCode

#### TypeScript code being linted

```ts
  const key = {
    accountType: '3B',
    uid: '456',
    isEditable: true,
  };
```

with `tslint.json` configuration:

```json
{ "object-literal-sort-keys": true }
```

#### Actual behavior
No error

#### Expected behavior
```The key 'isEditable' is not sorted alphabetically```

Downgrading to version 5.7.0 fixes the issue. This code succeeds on a Mac with 5.8.0.  It's possible there's something else at play besides Mac vs. Windows here but that's the only difference we've been able to find. Sticking with 5.7.0 for now.

**Comment by ajafff** [comment]
#3191 added the ability to group your properties by a blank line.
This probably introduced a bug where `\r\n` is treated as two line breaks on windows.

**Comment by ajafff** [comment]
#3427 will fix this
